autogpt_platform/frontend/src/components/nav/MarketPopup.tsx

```diff
@@ -7,7 +7,9 @@ interface MarketPopupProps extends ButtonHTMLAttributes<HTMLButtonElement> {
 
 export default function MarketPopup({
   className = "",
-  marketplaceUrl = "http://platform.agpt.co/marketplace",
+  marketplaceUrl = process.env.APP_ENV === "prod"
+    ? "https://production-marketplace-url.com"
```

Comment:
can we keep the existing link for prod and add  a spot for one for "dev": https://dev-builder.agpt.co/marketplace and finally have the localhost fallback for local